experiment_buddy/utils.py

```diff
@@ -43,10 +48,18 @@ def get_project_name(git_repo: git.Repo) -> str:
 def telemetry(f):
     def wrapped_f(*args, **kwargs):
         tic = time.perf_counter()
-        retr = f(*args, **kwargs)
+        should_exit = False
```

Comment:
```
        try:
            retr = f(*args, **kwargs)
        except StopProcess:
            toc = time.perf_counter()
            method_name = f.__name__
            requests.get(f"http://65.21.155.92/{method_name}/{toc - tic}", timeout=2.)
            sys.exit()
        else:
            toc = time.perf_counter()
            method_name = f.__name__
            requests.get(f"http://65.21.155.92/{method_name}/{toc - tic}", timeout=2.)
        return retr
```

this is what you mean?

Reply:
This would intertwine the logger with the whole system, the logger and the deploy shouldn't be aware one of the other 

Another approach would be to use a logger that uses the `with` statement instead of being a decorator and wrap everything but the `sys.exit()` to log the time, although this is way less AOP